examples/package.json

```diff
@@ -9,8 +9,8 @@
     "decodeData": "node src/decodeData.js"
   },
   "dependencies": {
-    "@erc725/erc725.js": "^0.14.0-beta.0",
-    "web3": "4.0.2"
+    "@erc725/erc725.js": "^0.17.2",
+    "web3": "^1.10.0"
```

Comment:
Why downgrade web3 ? I think ultimately we will want to have v4 ? (with ts support etc.)

Reply:
I wanted to keep the same version between `example` folder and the root package